content/events/2020-guadalajara/sponsor.md

```diff
@@ -3,64 +3,187 @@ Title = "Sponsor"
+  <div class="row justify-content-start">
+    <div class="col-md-9">
+      <div>
+      <h3></h3>
+      <p>We greatly value sponsors for this open event! If you are interested in sponsoring see a <a href="/events/2020-guadalajara/Prospectus.pdf">PDF of our prospectus</a> for more details or please drop us an email at {{< email_organizers >}}.</p>
+      <h1></h1>
+      <h1></h1>
 
-We greatly value sponsors for this open event.  If you are interested in sponsoring, please drop us an email at [{{< email_organizers >}}].
+      </div>
+      <h3>Sponsorship Packages</h3>
+      <div class="table-responsive">
+      <table class="table table-bordered table-hover table-responsive-md">
+        <thead class="thead-light">
+          <tr>
+            <th scope="col">
+              <i>Packages</i>
+            </th>
+            <th scope="col">
+              <center>Community</center>
+            </th>
+            <th scope="col">
+              <center>Bronze</center>
+            </th>
+            <th scope="col">
+              <center>Silver</center>
+            </th>
+            <th scope="col">
+              <center>Gold</center>
+            </th>
+            <th scope="col">
+              <center>Platinum</center>
+            </th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <th scope="row">Logo on website</th>
+            <td>
+              <center>Yes</center>
+            </td>
+            <td>
+              <center>Yes</center>
+            </td>
+            <td>
+              <center>Yes</center>
+            </td>
+            <td>
+              <center>Yes</center>
+            </td>
+            <td>
+              <center>Yes</center>
+            </td>
+          </tr>
+          <tr>
+            <th scope="row">Social media shout-out</th>
+            <td>
+              <center>Yes</center>
+            </td>
+            <td>
+              <center>Yes</center>
+            </td>
+            <td>
+              <center>Yes</center>
+            </td>
+            <td>
+              <center>Yes</center>
+            </td>
+            <td>
+              <center>Yes</center>
+            </td>
+          </tr>
+          <tr>
+            <th scope="row">Booth space</th>
+            <td>
+              <center> </center>
+            </td>
+            <td>
+              <center> </center>
+            </td>
+            <td>
+              <center>Shared</center>
+            </td>
+            <td>
+              <center>Own</center>
+            </td>
+            <td>
+              <center>Own + Premium placement</center>
+            </td>
+          </tr>
+          <tr>
+            <th scope="row">Full audience pitch</th>
+            <td>
+              <center> </center>
+            </td>
+            <td>
+              <center> </center>
+            </td>
+            <td>
+              <center> </center>
+            </td>
```

Comment:
Same as above - how about leaving the cell empty? e.g.
```
<td></td>
```

Reply:
oks used "empty cell" instead